### DISCUSSION TITLE
Object linking format in requests is not consistent

### DISCUSSION THREAD

**Discussion by weidox** [strapi/strapi#7050] [Legacy Topics]
When updating linked files in array, I can write linked file both as simple id string or as an object with "id" property, both ways are working in JSON requests:
"linkedfiles": [ "5f0d63ba5095242e24d30e80" ]
"linkedfiles": [ { "id": "5f0d63ba5095242e24d30e80"} ]

But when updating object property, only simple way is working:
{ "parent": "5f0d63ba5095242e24d30e80" } 
And this one does not work:
{ "parent": { "id": "5f0d63ba5095242e24d30e80" } 

Strapi 3.0.5

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
